From 7efcd5064f586bc888289deb0a5f288fdfe83994 Mon Sep 17 00:00:00 2001
From: YAMAMOTO Koichi <yamamoto@lineo.co.jp>
Date: Fri, 12 Nov 2021 17:06:19 +0900
Subject: [PATCH] Issue with rpath including /usr/lib and crosscompile checkes
 causing oe configure to fail

Use oe's cmake rpath framework and exclude some of the cmake checks that fail in our env

Upstream-Status: Inappropriate [configuration]
Singed-off-by: Armin Kuster <akuster808@gmail.com>

---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cf305d9..f9ee5aa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -434,14 +434,8 @@ include(TestInline)
 include(CheckFileOffsetBits)
 # Determine how to pack structs on this platform.
 include(CheckStructPacking)
-# Check for signed right shift implementation.
-include(CheckSignedRightShift)
 # Check if systtem fts implementation available
 include(CheckFTS)
-# Check if uname(2) follows POSIX standard.
-include(CheckUnamePosix)
-# Check support for file descriptor passing
-include(CheckFDPassing)
 
 # Check if big-endian
 include(TestBigEndian)
